src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -19,6 +19,7 @@ import * as config from '../../../config'
 import Stripe from 'stripe'
 import HTTPSProxyAgent from 'https-proxy-agent'
 import { format } from './csv'
+const { services } = require('../../../config')
```

Comment:
It's not required as toolbox still has some JavaScript left but for consistency with this file this would prefer the import syntax
```suggestion
import { services } from '../../../config'
```

Reply:
Done